js/lib/comments.js

```diff
@@ -113,6 +113,10 @@ elgg.comments.init = function() {
 		dc.toggleEdit();
 		return false;
 	});
+
+	// Highlight a comment if its id is found from the URL
+	var commentId = elgg.getSelectorFromUrlFragment(document.URL);
+	$(commentId).addClass('elgg-highlight').delay(1000).removeClass('elgg-highlight', {duration: 1000});
```

Comment:
I tried $.highlight but (at least with the basic settings) it didn't really do what I wanted. I could try it out a bit more, though.

What would be the benefit of CSS transition?